src/Validators/ImmunizationValidator.php

```diff
@@ -21,5 +23,21 @@ class ImmunizationValidator extends BaseValidator
     protected function configureValidator()
     {
         parent::configureValidator();
+
+        // insert validations
+        $this->validator->context(
```

Comment:
Did this get leaked in from another PR?  Not sure how its connected to this work.